cloud/deployment/deployment.go

```diff
@@ -274,105 +291,275 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeplo
-		DeploymentSpec:        spec,
-		APIKeyOnlyDeployments: *enforceCD,
+	createDeploymentRequest := astroplatformcore.CreateDeploymentRequest{}
+	// build standard input
+	if IsDeploymentStandard(deploymentType) || IsDeploymentDedicated(deploymentType) {
+		var highAvailabilityValue bool
+		if highAvailability == enable { //nolint: goconst
+			highAvailabilityValue = true
+		} else {
+			highAvailabilityValue = false
+		}
+		var workerConcurrency int
+		for i := range configOption.WorkerMachines {
+			if strings.EqualFold(configOption.DefaultValues.WorkerMachineName, configOption.WorkerMachines[i].Name) {
+				workerConcurrency = int(configOption.WorkerMachines[i].Concurrency.Default)
+			}
+		}
+		defautWorkerQueue := []astroplatformcore.WorkerQueueRequest{{
+			AstroMachine:      astroplatformcore.WorkerQueueRequestAstroMachine(configOption.DefaultValues.WorkerMachineName),
+			IsDefault:         true,
+			MaxWorkerCount:    int(configOption.WorkerQueues.MaxWorkers.Default),
+			MinWorkerCount:    int(configOption.WorkerQueues.MinWorkers.Default),
+			Name:              "default",
+			WorkerConcurrency: workerConcurrency,
+		}}
+		if defaultTaskPodCpu == "" {
+			defaultTaskPodCpu = configOption.ResourceQuotas.DefaultPodSize.Cpu.Default
+		}
+		if defaultTaskPodMemory == "" {
+			defaultTaskPodMemory = configOption.ResourceQuotas.DefaultPodSize.Memory.Default
+		}
+		if resourceQuotaCpu == "" {
+			resourceQuotaCpu = configOption.ResourceQuotas.ResourceQuota.Cpu.Default
+		}
+		if resourceQuotaMemory == "" {
+			resourceQuotaMemory = configOption.ResourceQuotas.ResourceQuota.Memory.Default
+		}
+		if IsDeploymentStandard(deploymentType) {
+			var requestedCloudProvider astroplatformcore.CreateStandardDeploymentRequestCloudProvider
+			if cloudProvider == gcpCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderGCP
+			}
+			if cloudProvider == awsCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderAWS
+			}
+			if cloudProvider == azureCloud {
+				requestedCloudProvider = astroplatformcore.CreateStandardDeploymentRequestCloudProviderAZURE
+			}
+			var requestedExecutor astroplatformcore.CreateStandardDeploymentRequestExecutor
+			if executor == CeleryExecutor {
+				requestedExecutor = astroplatformcore.CreateStandardDeploymentRequestExecutorCELERY
+			}
+			if executor == KubeExecutor {
+				requestedExecutor = astroplatformcore.CreateStandardDeploymentRequestExecutorKUBERNETES
+			}
+			standardDeploymentRequest := astroplatformcore.CreateStandardDeploymentRequest{
+				AstroRuntimeVersion:  runtimeVersion,
+				CloudProvider:        &requestedCloudProvider,
+				Description:          &description,
+				Name:                 name,
+				Executor:             requestedExecutor,
+				IsCicdEnforced:       isCicdEnforced,
+				Region:               &region,
+				IsDagDeployEnabled:   dagDeployEnabled,
+				IsHighAvailability:   highAvailabilityValue,
```

Comment:
```suggestion
				IsHighAvailability:   highAvailability,
```

Reply:
highAvailability is enable or disable...we need to send true or false here right?

https://github.com/astronomer/astro-cli/pull/1425/files#diff-b53f56e788c7abab81b9b7b016e1df0a87b7232006674c79a03bb7356c66c2ccR305

Reply:
yes  highAvailabilityValue is a bool and highAvailability is a string